project.yml

```diff
@@ -0,0 +1,99 @@
+attributes:
+  ORGANIZATIONNAME: matrix.org
+configs:
+  Debug: debug
+  Release: release
+name: Riot
+settingGroups:
+  appDefault:
+    CURRENT_PROJECT_VERSION: 20200826151407
+    SWIFT_VERSION: 5.3.1
+    BUNDLE_DISPLAY_NAME: Element
+    APPLICATION_GROUP_IDENTIFIER: group.im.vector
+    BASE_BUNDLE_IDENTIFIER: im.vector.app
+    KEYCHAIN_ACCESS_GROUP: $(AppIdentifierPrefix)$(BASE_BUNDLE_IDENTIFIER).keychain.shared
+    LD_RUNPATH_SEARCH_PATHS: $(inherited) @executable_path/Frameworks @executable_path/../../Frameworks
+    DEVELOPMENT_TEAM: 7J4U792NQT
+
+    RIOT_PROVISIONING_PROFILE: f65e7447-b8a3-46cc-8fba-fa60e55e2511
+    RIOT_PROVISIONING_PROFILE_SPECIFIER: Vector App Store
+    NSE_PROVISIONING_PROFILE: 31dc9316-e029-47fd-81f5-778db07d76a2
+    NSE_PROVISIONING_PROFILE_SPECIFIER: "Vector NSE: App Store"
+    SHARE_EXTENSION_PROVISIONING_PROFILE: 1a3be143-50c7-4ae2-834e-00596a053141
+    SHARE_EXTENSION_PROVISIONING_PROFILE_SPECIFIER: "Vector Share Extension: App Store"
+    SIRI_INTENTS_PROVISIONING_PROFILE: 18a66f93-ffe1-4008-b343-58350cc65023
+    SIRI_INTENTS_PROVISIONING_PROFILE_SPECIFIER: "Vector Siri Intents: App Store"
+
+settings:
+  base:
+    IPHONEOS_DEPLOYMENT_TARGET: 11.0
+    MARKETING_VERSION: 1.1.3
+    CURRENT_PROJECT_VERSION: 1.1.3
+    SDKROOT: iphoneos
+    TARGETED_DEVICE_FAMILY: 1,2
+    ALWAYS_SEARCH_USER_PATHS: NO
+    CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED: YES
+    CLANG_CXX_LANGUAGE_STANDARD: gnu++0x
+    CLANG_CXX_LIBRARY: libc++
+    CLANG_ENABLE_MODULES: YES
+    CLANG_ENABLE_OBJC_ARC: YES
+    CLANG_WARN__DUPLICATE_METHOD_MATCH: YES
+    CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING: YES
+    CLANG_WARN_BOOL_CONVERSION: YES
+    CLANG_WARN_COMMA: YES
+    CLANG_WARN_CONSTANT_CONVERSION: YES
+    CLANG_WARN_DEPRECATED_OBJC_IMPLEMENTATIONS: YES
+    CLANG_WARN_DIRECT_OBJC_ISA_USAGE: YES_ERROR
+    CLANG_WARN_EMPTY_BODY: YES
+    CLANG_WARN_ENUM_CONVERSION: YES
+    CLANG_WARN_INFINITE_RECURSION: YES
+    CLANG_WARN_INT_CONVERSION: YES
+    CLANG_WARN_NON_LITERAL_NULL_CONVERSION: YES
+    CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF: YES
+    CLANG_WARN_OBJC_LITERAL_CONVERSION: YES
+    CLANG_WARN_OBJC_ROOT_CLASS: YES_ERROR
+    CLANG_WARN_RANGE_LOOP_ANALYSIS: YES
+    CLANG_WARN_STRICT_PROTOTYPES: YES
+    CLANG_WARN_SUSPICIOUS_MOVE: YES
+    CLANG_WARN_UNREACHABLE_CODE: YES
```

Comment:
It can be interesting in the future to put Clang warnings in a xcconfig file and import it in each target one or in the base one. An example of a xcconfig file compiling warnings here: https://github.com/jonreid/XcodeWarnings.